ENVIRONMENTVARS.md

```diff
@@ -20,9 +20,12 @@ If an environment variable is updated, make sure to restart the server for the a
-| ----------------- | ------ | ---------------------------- | ------------------------------------------------------------------------ |
-| `NYPL_HEADER_URL` | string | "https://ds-header.nypl.org" | The base URL of the NYPL environment-specific header and footer scripts. |
+| Variable                                 | Type   | Value Example                        | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| ---------------------------------------- | ------ | ------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `NYPL_HEADER_URL`                        | string | "https://ds-header.nypl.org"         | The base URL of the NYPL environment-specific header and footer scripts.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `NEXT_PUBLIC_CLOSED_LOCATIONS`           | string | "all;Library of the Performing Arts" | A semicolon-delimited list of strings. Include quotes around the string. All locations beginning with any string in this list will be removed from the list of request options in the `ElectronicDelivery`, `HoldRequest`, and `ItemTableRow` components. Currently used physical locations: `Schwarzman;Science;Library for the Performing Arts;Schomburg`. To close all locations, add `all`. This will also remove EDD as a request option, the 'Request' buttons, and also disable the hold request/edd forms. If `all` is not present, EDD and 'Request' buttons will still be available. |
```

Comment:
What does `NEXT_PUBLIC` indicate?

Reply:
That prefix allows env variables to be used on the client side.